recipe/build.sh

```diff
@@ -15,7 +15,11 @@ case "$target_platform" in
     linux-*)
         autoreconf -ivf
         export LDFLAGS="${LDFLAGS} -Wl,--as-needed"
-        ./configure --prefix=$PREFIX $DISABLES
+        if [[ ${cuda_compiler_version} != "None" ]]; then
+          ./configure --enable-cuda --prefix=$PREFIX $DISABLES
```

Comment:
I think there's a `--disable-cuda` and `--disable-nvml` flag above that needs to be removed in this case

Reply:
@kkraus14 Thanks for the feedback - I updated the recipe accordingly.